web-modules/javax-servlets-2/src/main/java/com/baeldung/setparam/LanguageServlet.java

```diff
@@ -0,0 +1,23 @@
+package com.baeldung.setparam;
+
+import java.io.IOException;
+import java.util.Locale;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.*;
+
+@WebServlet(name = "LanguageServlet", urlPatterns = "/setparam/lang")
+public class LanguageServlet extends HttpServlet {
+
+    private static final long serialVersionUID = 2923732283720972121L;
```

Comment:
Is this necessary? Can we remove it to not distract from the main content?

Reply:
I have removed the field